### PR TITLE
WIP: start fixing CI builds by collecting information

### PR DESCRIPTION
**Pull request by petr-tik**
Last commit doesn't build
error message: https://travis-ci.org/gnzlbg/cargo-asm/jobs/406536922

Compiling locally with rust nightly doesn't stumble over the error,
which suggests that it was a problem caused by rustc 1.25

Switch to nightly in CI and announce in Readme that only nightly is supported

Currently only linux - that's my only working machine

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +18,7 @@ matrix:
     script:
     - cargo clippy -- -D clippy-pedantic
   - env: TARGET=x86_64-unknown-linux-gnu
-    rust: 1.25.0
+    rust: nightly
```
Comment (gnzlbg) [inline .travis.yml]: The problem with switching to Rust nightly here is that the assembly of every single test changes, potentially, on every different nightly. 

A better solution could be to update this build bot to the latest stable and update the tests to that. But I haven't tried if the latest stable fixes things.

**Comment by gnzlbg** [comment]
I'm closing this since it has been inactive for a while. If that changes, let me know and I'll reopen it.

**Comment by petr-tik** [comment]
Hi, I have been travelling over the last 3 weeks. Thanks for clarifying the approach to sticking to a specific version of stable rust. 

keen to pick it up again, if that's ok?

**Comment by gnzlbg** [comment]
Sure! No stress, let me know if you need help or have questions!

**Comment by gnzlbg** [comment]
I don't have that much appveyor CI capacity available, and the commits here are blocking my personal queue and preventing me from working on other projects, so I am closing this PR for the time being to prevent that from happening. It should be easy for you to enable appveyor and travis-ci for your fork, and let me know here when the tests pass there (I'll reopen the PR then).

**Pull request closed without merging**